opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/test/java/de/adorsys/opba/protocol/xs2a/tests/e2e/wiremock/WiremockE2EXs2aProtocolTest.java

```diff
@@ -181,6 +181,86 @@ void testAccountsListWithConsentUsingRedirectWithComputedIpAddress() {
+
+        when()
+                .fintech_calls_list_accounts_for_anton_brueckner_ip_address_compute()
+                .and()
+                .user_anton_brueckner_provided_initial_parameters_to_list_accounts_with_dedicated_consent()
+                .and()
+                .open_banking_redirect_from_aspsp_ok_webhook_called();
+        then()
+                .open_banking_has_consent_for_anton_brueckner_account_list()
+                .open_banking_can_read_anton_brueckner_account_data_using_consent_bound_to_service_session();
+    }
+
+    @Test
+    void testAccountsListWithConsentUsingRedirectWithComputedIpAddressForUnknownUser() {
+        given()
+                .redirect_mock_of_sandbox_for_anton_brueckner_accounts_running()
+                .preferred_sca_approach_selected_for_all_banks_in_opba(Approach.REDIRECT)
+                .rest_assured_points_to_opba_server();
+
+        when()
+                .fintech_calls_list_accounts_for_anton_brueckner()
+                .and()
+                .unknown_user_provided_initial_parameters_to_list_accounts_with_all_accounts_consent()
+                .and()
+                .user_anton_brueckner_provided_initial_psu_id_parameter_to_list_accounts_with_all_accounts_consent_with_ip_address_check()
+                .and()
+                .open_banking_redirect_from_aspsp_ok_webhook_called();
+        then()
+                .open_banking_has_consent_for_anton_brueckner_account_list()
+                .open_banking_can_read_anton_brueckner_account_data_using_consent_bound_to_service_session();
+    }
+
+    @Test
+    void testAccountsListWithConsentUsingRedirectWithWrongIbans() {
+        given()
+                .redirect_mock_of_sandbox_for_anton_brueckner_accounts_running()
+                .preferred_sca_approach_selected_for_all_banks_in_opba(Approach.REDIRECT)
+                .rest_assured_points_to_opba_server();
+
+        when()
+                .fintech_calls_list_accounts_for_anton_brueckner()
+                .and()
+                .user_anton_brueckner_provided_initial_parameters_to_list_accounts_with_wrong_ibans()
+                .and()
+                .got_503_http_error();
+    }
+
+    @ParameterizedTest
+    @EnumSource(Approach.class)
+    void testTransactionsListWithConsentUsingEmbeddedForUnknownUser(Approach approach) {
+        given()
+                .embedded_mock_of_sandbox_for_max_musterman_transactions_running()
+                .preferred_sca_approach_selected_for_all_banks_in_opba(approach)
+                .rest_assured_points_to_opba_server();
+
+        when()
+                .fintech_calls_list_transactions_for_max_musterman()
+                .and()
+                .unknown_user_provided_initial_parameters_to_list_transactions_with_single_accounts_consent()
+                .and()
+                .user_max_musterman_provided_initial_psu_id_parameter_to_list_transactions_with_single_account_consent()
```

Comment:
Maybe 
`user_max_musterman_provided_psu_id_parameter_to_list_transactions_with_single_account_consent`
as it is not initial

Reply:
done